DEVELOPING_TUTORIALS.md

```diff
@@ -485,16 +485,25 @@ In `static/tutorials.json`, add a new key for your tutorial and fill in the appr
   "title": "Your short tutorial title",
   "description": "Your tutorial description",
   "lessons": [
-    "Title of 1st lesson",
-    "Title of 2nd lesson",
-    "Title of 3rd lesson"
+    {
+      "title": "Title of a coding lesson (either standard or requiring file upload)",
+      "types": ["code"]
```

Comment:
thinking through this, is it necessary to be an array since we are only showing one icon?

Reply:
we had talked about adding `file-upload` here in the future, which wouldn't affect the icon but would help Cypress figure out what test to run

Reply:
Ah yes.
Looking back at it seems to be a bit confusing though.
I think that we can assume `file-upload` lessons also include code, meaning we can simply either `code` or `file-upload` type on a lesson, which allows to use it as a visual hint in the UI and also cypress will be able to do the proper testing.

So effectively, we just need the property `type` and its values can be `text`, `multiple-choice`, `code` or `file-upload`.

Sounds good?

Reply:
yep - i can refactor to make this work

Reply:
done in https://github.com/ProtoSchool/protoschool.github.io/pull/346/commits/852f2c7fa586b3815d995c9c386c4eeef0e91fe6 - note that this will also need to be fixed in your new multiple-choice tutorial